_posts/blog/2018-10-11-4rios-review.md

```diff
@@ -0,0 +1,30 @@
+
+On the opening webpage of "4 Ríos", the user's inital reaction is that of curious confusion, since the archive is not immediately recognizable as such. One might feel disillusioned by prior experiences with prototyical digital collections, library-like in their organization, as one attempts to master the interface and search for specific documentation. A visitor to "4 Ríos" transforms their experience when they agree to surrender instead, becoming a spectator as the narrative unfolds in graphic novel style, subject to the creators' conceived succession of information flow. If my description seems beyond grasp, it is because one must participate in this interactive historical retelling to conceive of its form.
+
+"4 Ríos" is not a researcher's archive. It is an archive for the people. A participant is presented with a series of choices while exploring the collection - choices which determine which set of information will present itself. A click on the largest of circles (["4 Ríos: El Naya"](http://4rios.co/elnaya)) leads one to the digital graphic novel which benevolently contains a tutorial for the novice user. By arrowing forwards, sound, comic book-like art, and video clips chronologically narrate the events surrounding what came to pass that April 11 of 2001 in Naya. A meta-index below the narration provides documented information on details surrounding the events with links to official documents online which are revealed as the participant progresses in the historical telling. The compilation is beautiful in its execution and harrowing in its narration. As the user surrenders to its innovative style, one is split both by awe at the compelling experience and by angry sadness at the murderous events.
+
+A tap to the smaller circle at the left (["Flujo de Memorias"](http://4rios.co/elnaya/flujo)) brings one into a vibrant forum of expression, where a participant can leave thoughts for future viewers by drawing a picture, or leaving a written or recorded message. A swipe of the mouse reveals that the colorful dots flowing across the stream are the river of memories left by past users, and exploring these fragments grants another dimension to archival information. To the right ([" Plataformas y Expansiones"](http://4rios.co/blog/expansiones-y-plataformas/)), the visitor can read about further projects for dissemination of this information, including access to the printed version of the graphic novel. A menu bar on the top gives options to visit the (["4 Ríos" blog](http://4rios.co/blog)), team information (["Equipo"](http://4rios.co/blog/equipo)), information on the project's production (["¿Qué es 4 Ríos"](http://4rios.co/blog/proyecto)) which leads to yet another page with more in-depth options, and recognitions (["Reconocimientos"](http://4rios.co/blog/reconocimientos)).
```

Comment:
I'm not sure we need a detailed description of each tab. Instead, consider describing more generally the ways in which this site does or does not serve different kinds of users. As a scholar, what kind of information can I find here? What's missing? What if I'm not a scholar - is this archive useful for other kinds of audiences?